src/pages/People/PeoplePage.tsx

```diff
@@ -0,0 +1,58 @@
+import { useContext, useEffect } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../../components/Loader';
+import { PeopleList } from '../../components/PeopleList/PeopleList';
+import { PeopleContext } from '../../peopleContext';
+
+export const PeoplePage = () => {
+  const { loader, warning, setLoader, setPeople, setWarning } = useContext(PeopleContext);
+
+  useEffect(() => {
+      getPeople()
+        .then(res => {
+          if (!res) {
+            setWarning('There are no people on the server');
+          } else {
+            setWarning('');
+          }
+
+          setPeople([...res]);
+        })
+        .catch(error => {
+          setWarning('Something went wrong');
+          throw error;
+        })
+        .finally(() => {
+          setLoader(false);
+        });
+  }, []);
```

Comment:
Fix it 
<img width="1190" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/64750958/e5b6fee4-5da8-4725-9ae8-06517e5e2389">